### DISCUSSION TITLE
秘境PK真的很烦

### DISCUSSION THREAD

**Discussion by Xm798** [Arondight/Adachi-BOT#709] [General]
米哈游把这什么鬼的秘境PK活动放官方资讯里，怕不是有什么大病，不到24小时就三条了，妥妥的扰民……

**Comment by Arondight** [comment]
有三种新闻，改改代码把资讯那种禁掉吧

**Reply by Xm798** [reply]
资讯也有有用的，比如活动玩法说明，角色EP，新版本一图流信息，角色生日之类的，之前都很正常，就是这个活动跟有大病一样

**Comment by mark9804** [comment]
大概是这样

https://github.com/Arondight/Adachi-BOT/blob/9bc2eebdb04222857ae1cb77e1b5efeced9215d2/src/jobs/news.js#L105

```js
const message = n.filter((c) => "string" === typeof c && "" !== c && !c.includes("PK")).join("\n");
```

**Reply by mark9804** [reply]
不对，当我没说

**Reply by Arondight** [reply]
https://github.com/Arondight/Adachi-BOT/blob/9bc2eebdb04222857ae1cb77e1b5efeced9215d2/src/jobs/news.js#L86

**Reply by mark9804** [reply]
```js
const postContent = post.content || "";
if (false === silent[t] && stamp > timestamp && !postContent.includes("PK")) {
```
这下应该对了
